app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -159,7 +159,10 @@ fun EditBillAmounts() {
                         .width(100.dp)
                         .height(55.dp),
                     textStyle = TextStyle.Default.copy(fontSize = 18.sp))
-                SaveRemindButton(enabled = rentBill.isNotEmpty(), inputEdited = rentBillEdited)
+                SaveRemindButton(
+                    enabled = rentBill.isNotEmpty(),
+                    inputEdited = rentBillEdited
+                )
```

Comment:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.